documentation/asciidoc/computers/raspberry-pi/bcm2711-bootloader.adoc

```diff
@@ -394,6 +405,21 @@ Default: `0x0` (no USB debug messages enabled)
 
 === Configuration Properties in `config.txt`
 
+[[boot_ramdisk]]
+==== boot_ramdisk
+If this property is set to `1` then the bootloader will attempt load a ramdisk file called `boot.img` containing the boot file-system. Subsequent files (e.g. `start4.elf`) are read from the ramdisk instead of the original boot file-system.
```

Comment:
Perhaps it's worth linking one of these "boot file-system" phrases to https://www.raspberrypi.com/documentation/computers/configuration.html#boot-folder-contents ?

Reply:
Sounds like a decent plan. Could you go ahead and do that. Either as a PR, or directly on develop. Thanks!

Reply:
I tried doing it directly on `develop`, but I'd forgotten I can't do that because it's a protected branch! :laughing:

Reply:
Oh yeah, I have admin permissions, so I can do things on branch that nobody else can. Doh!